make/builder-shell.py

```
@@ -0,0 +1,123 @@
+    _ = parser.add_argument(
+        "--branch",
+        default=None,
+        help="Which branch of builder to use, defaults to the current checked out branch.",
+    )
+
+
+def command(args: Namespace):
+    branch = cast(str | None, args.branch)
+    if branch is None:
+        branch = (
+            subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"])
```

Ruff (0.14.5)

Starting a process with a partial executable path

(S607)